lib/happo/public/happo-styles.css

```diff
@@ -1,17 +1,27 @@
 body {
-  background-color: #f0f0f0;
-  font-family: helvetica, arial;
-  background: repeating-linear-gradient(
-    -45deg,
-    #f0f0f0,
-    #f0f0f0 10px,
-    #f4f4f4 10px,
-    #f4f4f4 20px
+  background-color: #ebebeb;
+  background-image: linear-gradient(
```

Comment:
A comment here explaining what the gradient(s) accomplish would be nice (I should have added one earlier on).